docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
     
 <!-- INTERACTIVE page step2.md END -->
 
+<!-- INTERACTIVE page step3.md START -->
+
+## Explore Traces Plugin
+
+Another method to explore traces within Grafana is via the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels. 
```

Comment:
Any sugestions for a different word besides "explore traces using Explore Traces"
```suggestion
Another method to explore traces within Grafana is using the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels. 
```